examples/heatmap_example.py

```diff
@@ -0,0 +1,63 @@
+"""The script that produces bristlecone_heatmap_example.png."""
+
+import numpy as np
+import cirq
+
+
+def bristlecone():
+    value_map = {
+        (qubit.row, qubit.col): np.random.random() for qubit in cirq.google.Bristlecone.qubits
+    }
+
+    heatmap = cirq.Heatmap(value_map)
+    file_path = "examples/bristlecone_heatmap_example.png"
+    fig, _, _ = heatmap.plot()
+    fig.figure.savefig(file_path)
+
+
+def gateerror():
+    title = 'Two Qubit Sycamore Gate Xeb Cycle Total Error'
+    value_map = {
+        (cirq.GridQubit(3, 2), cirq.GridQubit(4, 2)): 0.004619111460557768,
+        (cirq.GridQubit(4, 1), cirq.GridQubit(4, 2)): 0.0076079162393482835,
+        (cirq.GridQubit(4, 1), cirq.GridQubit(5, 1)): 0.010323903068646778,
+        (cirq.GridQubit(4, 2), cirq.GridQubit(4, 3)): 0.00729037246947839,
+        (cirq.GridQubit(4, 2), cirq.GridQubit(5, 2)): 0.008226663382640803,
+        (cirq.GridQubit(4, 3), cirq.GridQubit(5, 3)): 0.01504682356081491,
+        (cirq.GridQubit(5, 0), cirq.GridQubit(5, 1)): 0.00673880216745637,
+        (cirq.GridQubit(5, 1), cirq.GridQubit(5, 2)): 0.01020380985719993,
+        (cirq.GridQubit(5, 1), cirq.GridQubit(6, 1)): 0.005713058677283056,
```

Comment:
we should use a randomly generated data instead with a seed and then use the image generated based on that

Reply:
I am okay with using a randomly generated data. Any benefits as opposed to a fixed example?

Reply:
It's just more concise! The end result is the same - if you provide a seed.